Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -843,7 +843,7 @@ VideoConfigDiag::VideoConfigDiag(wxWindow* parent, const std::string& title)
                                             Config::GFX_CACHE_HIRES_TEXTURES);
       szr_utility->Add(cache_hires_textures);
 
-      szr_utility->Add(CreateCheckBox(page_advanced, _("Full Resolution Frame Dumps"),
+      szr_utility->Add(CreateCheckBox(page_advanced, _("Internal Resolution Frame Dumps"),
```

Comment:
This should be changed in DolphinQt2 too:
https://github.com/dolphin-emu/dolphin/blob/6cfd7ba/Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp#L65